src/Microsoft.VisualStudio.Composition/StandardAssemblyLoader.cs

```diff
@@ -31,6 +31,19 @@ public Assembly LoadAssembly(AssemblyName assemblyName)
 
             if (assembly == null)
             {
+#if NETCOREAPP1_0
+                // Clone the AssemblyName to undefine the CodeBase property in case it is set.
+                // Workaround for https://github.com/dotnet/coreclr/issues/10561
+                assemblyName = new AssemblyName
```

Comment:
nit: consider doing the same method as the fix for that issue here (see https://github.com/dotnet/coreclr/pull/11010/files#diff-941b565c09cd92cf3c9588857554f825R108).